libc/src/math/CMakeLists.txt

```diff
@@ -448,6 +448,8 @@ add_math_entrypoint_object(tanhf)
 
 add_math_entrypoint_object(tgamma)
 add_math_entrypoint_object(tgammaf)
+add_math_entrypoint_object(lgamma)
+add_math_entrypoint_object(lgamma_r)
```

Comment:
Nit: only type variants of the same function are grouped together (except a few exceptions that should be fixed), so `tgamma*`, `lgamma*` and `lgamma_r*` should be in three separate groups. Definitions are also sorted alphabetically, by function prefix/base name without type suffix.